test/support/router.ex

```diff
@@ -49,6 +49,8 @@ defmodule DrabTestApp.Router do
     get("/tests/live/broadcasting", LiveController, :broadcasting, as: :broadcasting)
 
     get("/tests/element", ElementController, :index, as: :element)
+    
+    get("/tests/pubsub", PubsubController, :index, as: :pubsub)
```

Comment:
Drab already has pubsub functionality, although it's tested in the `LiveController` test.  It really should be pulled out into it's own test, and `Drab.Channel`'s message receiver should be changed so if the topic starts with `drab:` then it should do as it does now, else it should call a callback in the appropriate commander (which could potentially be shared commanders).

Reply:
I don't understand this comment. Here we are just testing the new added functionalities with a PubSub lifecycle case, thus the name 'pubs' for this test.

Reply:
Oh I was just saying that the little bit of pubsub use that drab uses is testing in the `/tests/live/broadcasting` route, when they should probably be pulled out into their own.  ^.^